Sources/SwiftSyntax/Syntax.swift

```diff
@@ -483,6 +489,12 @@ public extension SyntaxProtocol {
   /// The trailing trivia of this syntax node. Trailing trivia is attached to
   /// the last token syntax contained by this node. Without such token, this
   /// property will return nil.
+  ///
+  /// Note: `Trivia` is not able to represent invalid UTF-8 sequences. To get
+  /// the leading trivia text including all invalid UTF-8 sequences, use
+  /// ```
+  /// node.syntaxTextBytes[(node.byteSize - node.trailingTriviaLength.utf8Length)...]
```

Comment:
Maybe
```swift
node.syntaxTextBytes.suffix(node.trailingTriviaLength.utf8Length)
```
looks easier. But it doesn't really matter

Reply:
Actually, there is no `suffix` on `Array`. But I took the suggestion for `prefix`.